.env.example

```diff
@@ -1,25 +1,6 @@
 # ~~
 # values for elm-pages front end
 # ~~
-
-CANONICAL_URL=http://localhost:3030/
-PLACECAL_API=http://localhost:3000/api/v1/graphql
-JOIN_US_FUNCTION_URL=http://localhost:8888/.netlify/functions/transDim
-
-
-
-# ~~
-# values for netlify backend
-# ~~
-
-EMAIL_ADMIN=user@gfsc.studio
-EMAIL_FROM=user@gfsc.studio
-EMAIL_INFO_TRANS_DIM=user@gfsc.studio
-
-# email server
-SMTP_HOST=127.0.0.1
-SMTP_PORT=1025
-SMTP_PASSWORD=
-SMTP_USERNAME=
-SMTP_SECURE=
-SMTP_REQUIRE_TLS=
\ No newline at end of file
+CANONICAL_URL=http://localhost:9000/
+PLACECAL_API=http://0.0.0.0:3000/api/v1/graphql
```

Comment:
Why have we switched from `localhost` to `0.0.0.0`? I also, personally, think this should go back to being live PlaceCal, so you don't have to have PlaceCal running locally by default to get events.

Reply:
I've made this change, it seems sensible for someone who might want to submit to not be encouraged to install placecal as well.

Reply:
`0.0.0.0` is useful as it means you can test your local dev server on another machine on your network. localhost only allows same-host connections.

Could put in a comment that the production/staging URLs are such and such. This is just a template for the dev to customise.